packages/server/database/rethinkDriver.ts

```diff
@@ -91,7 +91,7 @@ export type RethinkSchema = {
   }
   NewMeeting: {
     type: AnyMeeting
-    index: 'facilitatorUserId' | 'teamId' | 'templateId'
+    index: 'facilitatorUserId' | 'teamId' | 'templateId' | 'scheduledEndTime' | 'hasNotEnded'
```

Comment:
-1 should be `hasEndedScheduledEndTime` (looks like the `between` method in stricterR isn't strict, so it didn't catch this one)